pkgs/applications/graphics/ktikz/default.nix

```diff
@@ -0,0 +1,74 @@
+{ withKDE ? true,
```

Comment:
It would also work of course, but it seems a bit clearer to me. Configuring a package by passing `null`'s is messy.